core/src/com/g16/feyrune/map/Map.java

```diff
@@ -9,7 +9,7 @@ public class Map {
     private final int width;
     private final int height;
 
-    protected Map(Tile[][] tiles) {
+    public Map(Tile[][] tiles) {
```

Comment:
Was it impossible to have a less open "privacy setting" (don't remember the correct term)

Reply:
You are correct, didnt move everything to the right position